texts/epn_paper.md

```diff
@@ -0,0 +1,67 @@
+# Let's talk online
+
+<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->
+
+Your routine changed drastically over the course of the last couple of months due to the lockdowns. You now mostly stay at home, hopefully safe and healthy.
+Your classes are now online, and probably you had to quickly figure out how run them at all.
+If you do experiments, your lab is either closed or operates at a much lower capacity.
+Your travel plans are cancelled. Still, despite that, you have likely attended a scientific presentation or even gave one yourself.
+The only difference is that it happened online.
```

Comment:
The only difference is that it happened online, perhaps as part of a virtual conference.